x-pack/plugins/alerting/server/lib/rule_execution_status.ts

```diff
@@ -107,10 +107,6 @@ export function ruleExecutionStatusFromRaw(
     executionStatus.lastDuration = lastDuration;
   }
 
-  if (null != numberOfTriggeredActions) {
-    executionStatus.numberOfTriggeredActions = numberOfTriggeredActions;
-  }
-
```

Comment:
I think `numberOfTriggeredActions` is a leftover here... We don't keep `numberOfTriggeredActions` in SO. So the above if condition can never be true.